HousingSearchApi.Tests/V1/Factories/TenureQueryGeneratorTests.cs

```diff
@@ -0,0 +1,165 @@
+using AutoFixture;
+using Hackney.Core.ElasticSearch.Interfaces;
+using Hackney.Shared.HousingSearch.Gateways.Models.Tenures;
+using HousingSearchApi.V1.Boundary.Requests;
+using HousingSearchApi.V1.Infrastructure.Factories;
+using HousingSearchApi.V1.Interfaces;
+using Moq;
+using Nest;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace HousingSearchApi.Tests.V1.Factories
+{
+    public class TenureQueryGeneratorTests
+    {
+        private readonly TenureQueryGenerator _sut;
```

Comment:
What does 's' stand for within `_sut`?

Reply:
I followed the existing naming convention in these classes, but arguable system under test might not be the most appropriate term for this scope. I left it as is for consistency.